### ISSUE TITLE
The 06_validate_datasetjson.sas program (potentially underlying code) appears to truncate the message to 250 characters

### ISSUE THREAD

**Issue by mhungria** [lexjansen/dataset-json-sas#20]
he 06_validate_datasetjson.sas program (or underlying code) appears to truncate the message to 250 characters:

'1' is not of type 'number' Failed validating 'type' in schema['properties']['clinicalData']['properties']['itemGroupData']['additionalProperties']['properties']['itemData']['items']['items'][0]: {'description': 'The first item in the data array need

**Comment by lexjansen** [comment]
This might be a limitation of SAS PROC FCMP. When you run the code in Python (see scripts/json_validate.py) it will dump the error message and the complete Dataset-JSON file. I will look into this. Also, when there are multiple issues, it would be nice to see them all.

**Comment by mhungria** [comment]
Yes, I was going to report that separately as it appears it is an independent issue of just reporting the first issue it encounters on each file.

**Comment by lexjansen** [comment]
This is fixed. PROC FCMP can only return 255 characters on a line. Instead of dumping the whole exception heap, you will get in the results dataset: (as an example):
- result_code = 1
- result_character = "'1' is not of type 'number' (The first item in the data array needs to be a number (itemGroupDataSeq))"
- result_path = "$.clinicalData.itemGroupData.IG.RELREC.itemData[0][0]"

I will open a seperate enhancement request to get all errors.

**Comment by mhungria** [comment]
Tested the fix. Looks ok now.

Looking forward to test the enhancement #22 when implemented.
